.github/workflows/release-baseline-images.yml

```
@@ -0,0 +1,39 @@
+# Upload the ZIP archive of baseline images as a release asset
+
+name: Upload baseline images
+
+# Only run for releases
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  upload-baseline:
+    name: Upload baseline images
+    runs-on: ubuntu-latest
+    if: github.repository == 'GenericMappingTools/pygmt'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+
+    - name: Setup data version control (DVC)
+      uses: iterative/setup-dvc@v1.0.3
+
+    - name: Pull baseline image data from dvc remote
+      run: |
+        dvc pull
+        ls -lhR pygmt/tests/baseline/
+
+    - name: Create the baseline image asset in zip format
+      run: |
+        mkdir baseline-images
+        mv pygmt/tests/baseline/*.png baseline-images/
+        zip -r baseline-images.zip baseline-images
```

```suggestion
        zip -r baseline-images.zip baseline-images
        shasum -a 256 baseline-images.zip
```
It would be helpful to have the checksum reported for verification.